keps/sig-architecture/2140-receipts/README.md

```diff
@@ -0,0 +1,266 @@
+
+This is very time intensive for the release team and leads to many instances where notifications are missed, ultimately resulting in untracked Exceptions and rushed Exception Requests.
+
+During the implementation phase of the release, more comments and back and forth conversation occurs on the Enhancement issue, including:
+
+* tracking of kubernetes/kuberentes PRs[[1](https://github.com/kubernetes/enhancements/issues/1929#issuecomment-726228115)]
+* tracking kubernetes/website PRs[[2](https://github.com/kubernetes/enhancements/issues/2000#issuecomment-722752540)].
+
+This is also done simply by referencing the KEP from the kubernetes/kubernetes PR.
+
+When a milestone, such as enhancement freeze or code freeze occur, the issue is updated to reflect inclusion or removal from the milestone and we again leverage comments and manually applied labels to convey status, as an example consider [Build-in API Types Misses Code Freeze](https://github.com/kubernetes/enhancements/issues/1929#event-3991979549).
+
+Additionally, when an exception is required, this is [handled via email](https://groups.google.com/g/kubernetes-sig-release/c/eCk4Xyb58GQ/m/tjgEstjyAQAJ) and linked back to the issue via additional comments, such as [Built-in API Types Defaults Exception Granted](https://github.com/kubernetes/enhancements/issues/1929#issuecomment-727043537).
+
+Over time, this process becomes harder and harder to navigate, leading to decreased process clarity, leading to frustration by contributors and increased toil for the release team.
+
+In order to streamline this process and increase transparency, we will add a new git based artifact, hereafter referred to as a receipt. This receipt represents an opt-in promise/soft commitment from a SIG about KEPs they’re submitting to a specific release and will allow all tracking to occur via Git commits, rather than via issue comments, PR references and other non auditable mechanisms.
+
+### Goals
+
+* Support better KEP validation and automation through Git based process
+* Reduce friction and toil early in a release cycle
+* Eliminate instances where Enhancements aren't tracked due to miscommunication
+* Encourage SIGs to proactively update KEPs and plan for release cycles
+* Improve Exception process turning exception requests into a PR by the SIG with proper approvals
+* Increase KEP / Release transparency
+* Encourage better KEP implementation history updates
+
+
+### Non-Goals
+
+<!--
+What is out of scope for this KEP? Listing non-goals helps to focus discussion
+and make progress.
+-->
+
+
+## Proposal
+
+The process for opting in to a release will accept a specified file format (hereafter referred to as a `receipt`) containing metadata pertaining the KEP being enrolled. There would be one receipt per KEP per release cycle.
```

Comment:
As written the process itself is accepting the specified file format — I think you mean the tool or person(s)